Examples/Examples-iOS/IGListKitExamples/ViewControllers/SupplementaryViewController.swift

```diff
@@ -20,17 +20,50 @@ final class SupplementaryViewController: UIViewController, ListAdapterDataSource
     lazy var adapter: ListAdapter = {
         return ListAdapter(updater: ListAdapterUpdater(), viewController: self)
     }()
-    let collectionView = UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewFlowLayout())
+    let collectionView = UICollectionView(frame: .zero, collectionViewLayout: ListCollectionViewLayout(
+        stickyHeaders: true, topContentInset: 0.0, stretchToEdge: false))
 
     let feedItems = [
-        FeedItem(pk: 1, user: User(pk: 100, name: "Jesse", handle: "jesse_squires"), comments: ["You rock!", "Hmm you sure about that?"]),
-        FeedItem(pk: 2, user: User(pk: 101, name: "Ryan", handle: "_ryannystrom"), comments: ["lgtm", "lol", "Let's try it!"]),
-        FeedItem(pk: 3, user: User(pk: 102, name: "Ann", handle: "abaum"), comments: ["Good luck!"]),
-        FeedItem(pk: 4, user: User(pk: 103, name: "Phil", handle: "phil"), comments: ["yoooooooo", "What's the eta?"])
+        FeedItem(pk: 1, user: User(pk: 100, name: "Jesse", handle: "jesse_squires"), comments: [
+            "You rock!",
+            "Hmm you sure about that?",
+            "You rock!",
+            "Hmm you sure about that?",
+            "You rock!",
+            "Hmm you sure about that?"
+            ]),
+        FeedItem(pk: 2, user: User(pk: 101, name: "Ryan", handle: "_ryannystrom"), comments: [
+            "lgtm",
+            "lol",
+            "Let's try it!",
+            "lol",
+            "Let's try it!",
+            "Good luck!"
+            ]),
+        FeedItem(pk: 3, user: User(pk: 102, name: "Ann", handle: "abaum"), comments: [
+            "Good luck!",
+            "yoooooooo",
+            "lol"
+            ]),
+        FeedItem(pk: 4, user: User(pk: 103, name: "Phil", handle: "phil"), comments: [
+            "yoooooooo",
+            "What's the eta?"
+            ])
     ]
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        if let flowLayout = collectionView.collectionViewLayout as? ListCollectionViewLayout {
+            let statusBarSize = UIApplication.shared.statusBarFrame.size
+            let navigationBarVisible = self.navigationController?.isNavigationBarHidden == false
+            let navigationBarHeight = ( navigationBarVisible ? self.navigationController!.navigationBar.frame.height : 0 )
```

Comment:
@rnystrom Sure, I will see how I can change the code to make it work without accessing the nav/status